examples/tests/test_examples.py

```diff
@@ -148,7 +148,7 @@ def test_examples_compare(filename, pytestconfig, force_offscreen, mock_time):
     stored_img = iio.imread(screenshot_path)
 
     # assert similarity
-    atol = 2
+    atol = 1
```

Comment:
While I thought that we could decrease this tolerance, it seems clear to me that some of the errors are mostly introduced due to the rounding of the atlas sampler.

```suggestion
    atol = 2
```

If you agree, you can commit and get this pushed through.

Reply:
Well, we have not updated the reference screenshots yet.

Reply:
I have a few times. Always reverted since it seemed like it wasn’t helping. 

You can use the pixel peeper application to see that the errors are coming from the middle of the letter k and this is not something that is thought to be affected by this PR. At least in theory.